### ISSUE TITLE
RMLSEND AL error

### ISSUE THREAD

**Issue by Chester-Graham** [ElvishArtisan/rivendell#564]
I am running Rivendell 3.3.0 on Centos 7
I am running the RMLSend Macro Command Utility Gui
I have 2 logs currently on my system the first is named SAMPLE, the second is named Production-0420
If I LL 1 SAMPLE ! I get the log for SAMPLE log loaded.
If I then do AL 1 Production-0420 ! I get another copy of SAMPLE appended to the currently running log.
If I reverse this and LL 1 Production-0420 ! it loads the correct log,
If I do AL 1 SAMPLE !, I get another copy of Production-0420 appended.

In summary, AL seems to ignore the log to load, and load another copy of the currently loaded log.

**Comment by ElvishArtisan** [comment]
Confirmed here.

**Comment by ElvishArtisan** [comment]
Fixed in 118c019.
